cla_public/templates/contact.html

```diff
@@ -154,26 +154,23 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
   </script>
 
   <script type="text/html" id="postcodeNotEnteredText">
-    <div class="form-row field-error">
-      <p>
-        {{ _('Must contain a valid postcode') }}
-      </p>
-    </div>
+    <span class="govuk-error-message">
+      <span class="govuk-visually-hidden">{% trans %}Error{% endtrans %}:</span>
+      <span class="cla-error-message">{{ _('Must contain a valid postcode') }}</span>
```

Comment:
Why the switch between {% trans %} and {{ _('') }}

Reply:
No reason.  The {{ _('') }} approach was already there - I just added stuff around it.  
The {% trans %} approach was copied from elsewhere.

Reply:
In that case, can we make them consistent for the parts we are updating, so maybe replace the  {{ _('') }} with trans

Reply:
Changed to `{{ _('') }}`